### PR TITLE
Docs  - Dynatrace webhook

### PR DESCRIPTION
**Pull request by PeyGis**
# Description

A webhook integration between Port and Dynatrace. This doc guides users to ingest problem notifications from Dynatrace entities to Port.

## Added docs pages

Please also include the path for the added docs.

{{BASE_URL}}/build-your-software-catalog/sync-data-to-catalog/webhook/examples/dynatrace

### CODE REVIEW

**Comment by PeyGis** [comment]
image

image

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/dynatrace/_example_dynatrace_problem_webhook_configuration.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      "blueprint": "dynatraceProblem",
+      "entity": {
+        "identifier": ".body.ProblemID | tostring",
+        "title": ".body.ProblemTitle",
+        "properties": {
+          "state": ".body.State",
+          "url": ".body.ProblemURL",
+          "details": ".body.ProblemDetailsText",
+          "impact": ".body.ProblemImpact",
+          "severity": ".body.ProblemSeverity"
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/dynatrace/_example_dynatrace_problem_webhook_configuration.mdx:18]: Please check if you can also map the problem tags to an array of tags (you will also need to update the dynatrace problem blueprint with this new property)

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/dynatrace.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<details>
+<summary>Dynatrace problem blueprint</summary>
+<DynatraceProblemBlueprint/>
+</details>
+
+<details>
+<summary>Dynatrace problem webhook configuration</summary>
+<DynatraceProblemConfiguration/>
+</details>
+
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/dynatrace.md:32]: Add here an [admonition](https://docusaurus.io/docs/markdown-features/admonitions) block of type note that says that the relation mapping in the webhook configuration will only work if the identifiers of Port microservice entities matches the names of entities in your dynatrace

**Review by MPTG94 — CHANGES_REQUESTED** [review]
Please fix the comments and suggestions I added

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/dynatrace/_example_dynatrace_problem_webhook_configuration.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        },
+        "relations": {
+          "microservice": ".body.ImpactedEntities[].entity"
```
[the remark this answers is not in the document]
Comment (PeyGis) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/dynatrace/_example_dynatrace_problem_webhook_configuration.mdx:21]: Thanks for this. I had asked about it in the channel

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/dynatrace/_example_dynatrace_problem_webhook_configuration.mdx (hunk 1 above)
Comment (PeyGis) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/resources/dynatrace/_example_dynatrace_problem_webhook_configuration.mdx:18]: I have used arrays for the `tags` property. Let me know if I should convert it to string, as that's the default type from the webhook
```{
  "Tags": "testtag1, testtag2"
}
```

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/dynatrace.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+:::note
+The relation mapping in the webhook configuration will only work if the identifiers of Port microservice entities matches the names of entities in your dynatrace.
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/dynatrace.md:34]:
```suggestion
The relation mapping in the webhook configuration will only work if the identifiers of Port microservice entities matches the names of entities in your Dynatrace.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/dynatrace.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+5. Select **Custom integration** from the available intregration types;
+6. Input the following details:
+   1. `Display name` - use a meaningful name such as Port Webhook;
+   2. `Webhook URL` - enter the value of the `url` key you received after creating the webhook configuration;
+   3. `Overview` - you can add an optional HTTP header to your webhook request;
+   4. `Custom payload` - When a problem is detected or resolved on your entity, this payload will be sent to the webhook URL. You can enter this JSON placeholder in the textbox;
+   ```json showLineNumbers
+   {
+   "State":"{State}",
+   "ProblemID":"{PID}",
```
[the remark this answers is not in the document]
Comment (PeyGis) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/dynatrace.md:52]: noted

**Pull request merged**